src/model/core/non_served_energy.jl

```diff
@@ -1,14 +1,32 @@
 @doc raw"""
     non_served_energy!(EP::Model, inputs::Dict, setup::Dict)
-This function defines the non-served energy/curtailed demand decision variable $\Lambda_{s,t,z} \forall s \in \mathcal{S}, \forall t \in \mathcal{T}, z \in \mathcal{Z}$, representing the total amount of demand curtailed in demand segment $s$ at time period $t$ in zone $z$. The first segment of non-served energy, $s=1$, is used to denote the cost of involuntary demand curtailment (e.g. emergency load shedding or rolling blackouts), specified as the value of $n_{1}^{slope}$. Additional segments, $s \geq 2$ can be used to specify a segment-wise approximation of a price elastic demand curve, or segments of price-responsive curtailable loads (aka demand response). Each segment denotes a price/cost at which the segment of demand is willing to curtail consumption, $n_{s}^{slope}$, representing the marginal willingness to pay for electricity of this segment of demand (or opportunity cost incurred when demand is not served) and a maximum quantity of demand in this segment, $n_{s}^{size}$, specified as a share of demand in each zone in each time step, $D_{t,z}.$ Note that the current implementation assumes demand segments are an equal share of hourly load in all zones.
-This function defines contributions to the objective function from the cost of non-served energy/curtailed demand from all demand curtailment segments $s \in \mathcal{S}$ over all time periods $t \in \mathcal{T}$ and all zones $z \in \mathcal{Z}$:
+This function defines the non-served energy/curtailed demand decision variable
+$\Lambda_{s,t,z} \forall s \in \mathcal{S}, \forall t \in \mathcal{T}, z \in \mathcal{Z}$,
+representing the total amount of demand curtailed in demand segment $s$ at
+time period $t$ in zone $z$.
+The first segment of non-served energy, $s=1$, is used to denote the cost of
+involuntary demand curtailment (e.g. emergency load shedding or rolling blackouts),
+specified as the value of $n_{1}^{slope}$.
+Additional segments, $s \geq 2$ can be used to specify a segment-wise approximation of a
+price elastic demand curve, or segments of price-responsive curtailable demands
```

Comment:
I don't like the phrase 'curtailable demand' but not sure of a better one.

Reply:
I'm okay with this, I think it's pretty standard terminology